gnes/service/frontend.py

```diff
@@ -139,13 +139,17 @@ def StreamCall(self, request_iterator, context):
             with self.zmq_context as zmq_client:
                 num_request = 0
                 for request in request_iterator:
-                    zmq_client.send_message(self.add_envelope(request, zmq_client), self.args.timeout)
+                    zmq_client.send_message(self.add_envelope(request, zmq_client), -1)
```

Comment:
isn't it the same thing as setting `timeout` to -1?

Reply:
for streaming mode, the `send_message` method often raise **timeout** exception (sometimes the socket is not free) when its argument `timeout` is enabled.

Reply:
hence, the `timeout` is only valid for receiving responses